src/features/lightspeed/lightSpeedOAuthProvider.ts

```diff
@@ -90,6 +93,14 @@ export class LightSpeedAuthenticationProvider
     );
   }
 
+  get optOutTelemetry() {
```

Comment:
This can be moved in the `LightSpeedManager` class. The `LightSpeedAuthenticationProvider` class methods can be limited to the authentication logic.

Reply:
Good point, thank you @ganeshrn, applied!